packages/synthetic-chain/upgrade-test-scripts/run_prepare.sh

```diff
@@ -23,8 +22,25 @@ grep -qF 'printKeys' /root/.bashrc || echo "printKeys" >>/root/.bashrc
 
 source ./env_setup.sh
 
+PROPOSAL=$1
+if [ -z "$PROPOSAL" ]; then
+  fail "Must specify what proposal to use"
+fi
+
 startAgd
 
+echo "[$PROPOSAL] Agd started."
+cd /usr/src/proposals/"$PROPOSAL/" || fail "Proposal $PROPOSAL does not exist"
+
+if [ -f "prepare.sh" ]; then
+  # In case the propsoal needs some chain actions to be done before the upgrade,
```

Comment:
```suggestion
  # In case the proposal needs some chain actions to be done before the upgrade,
```

Reply:
thanks. I'll land this since CI is slow slow and plan to do a spell check later